nox/virtualenv.py

```diff
@@ -39,6 +39,22 @@
 _SYSTEM = platform.system()
 
 
+def find_uv() -> str | None:
+    uv = shutil.which("uv")
+    if uv is not None:
+        return uv
+
+    # Handle `pipx install nox[uv]` by looking for uv next to python.
+    uv = os.path.join(os.path.dirname(sys.executable), "uv")
```

Comment:
I guess this needs `.exe` on Windows. Can also just use `shutil.which("uv", path=os.path.dirname(sys.executable))` which tries a bunch of different suffixes on Windows. [This](https://github.com/astral-sh/uv/blob/0bc047866d35c3944c27e71a185fae44e7ce3ae1/python/uv/__init__.py#L8-L30) is what uv itself does.

Reply:
Using `find_uv_bin` sounds perfect, thanks!